INZFS/Pages/EligibilityChecker.cshtml

```diff
@@ -10,33 +10,40 @@
     var listenabled = @eligibility.Content.EligibilityChecker.Bullet1?.Text == null;
     var AccordionListEnabled = @eligibility.Content.EligibilityChecker.AccordionBullet1?.Text == null;
     var titlePart = (string)@eligibility.Content.AliasPart.Alias;
+    var backLink = "/";
     var questionSettingA = "Question1";
     ViewData["Title"] = "Eligibility Checker";
     switch (titlePart)
     {
         case "Question1":
             questionSettingA = "Question1";
+            backLink = "/eligibilityInfo/Eligibility-Start";
             break;
         case "Question2":
             questionSettingA = "Question2";
+            backLink = "/eligibilityChecker/Question1";
             break;
         case "Question3":
             questionSettingA = "Question3";
+            backLink = "/eligibilityChecker/Question2";
             break;
         case "Question4":
             questionSettingA = "Question4";
+            backLink = "/eligibilityChecker/Question3";
             break;
         case "Question5":
             questionSettingA = "Question5";
+            backLink = "/eligibilityChecker/Question4";
             break;
         case "Question6":
             questionSettingA = "Question6";
+            backLink = "/eligibilityChecker/Question5";
             break;
         default: questionSettingA = "Question1"; break;
     }
 }
 
-<a href="#" class="govuk-back-link" id="govuk-back-link">Back</a>
+<a href=@backLink class="govuk-back-link" id="govuk-back-link">Back</a>
```

Comment:
Wonder if it should be within a double quote

Reply:
Changes applied